brainiak/utils/fmrisim.py

```diff
@@ -1524,48 +1882,77 @@ def mask_brain(volume,
     zoom_factor = (brain_dim[0] / mask_dim[0],
                    brain_dim[1] / mask_dim[1],
                    brain_dim[2] / mask_dim[2],
-                   1)
+                   )
 
     # Scale the mask according to the input brain
     # You might get a warning but ignore it
```

Comment:
Which part is throwing a warning? If it is not worth fixing, we should at least be more specific so that meaningful warnings don't get mistakenly ignored.

Reply:
The issue is when zoom factor values are non integers. I have checked that the functionality is still present even with the warnings so added a comment describing this and saying why they can be safely ignored